charts/seed-bootstrap/templates/priorityclass-shoot-controlplane.yaml

```diff
@@ -0,0 +1,8 @@
+---
+apiVersion: {{ include "priorityclassversion" . }}
+kind: PriorityClass
+metadata:
+  name: shoot-controlplane
```

Comment:
Hm, not sure, maybe we introduce other priority classes for the remaining control plane components in the future. This kind of hierarchy is more difficult instead of having a clear name for which the class gets used. We can't include the actual priority in the name anyways, hence, it is anyways needed to read the resource to get to know the priority value.

Reply:
Can we prefix with it `gardener`?

Reply:
Good suggestion, thanks, will do.